app/src/ui/keyboard-shortcut/keyboard-shortcut.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+
+interface IKeyboardShortCutProps {
+  /** Windows/Linux keyboard shortcut */
+  readonly keys: ReadonlyArray<string>
+  /** MacOS keyboard shortcut */
+  readonly darwinKeys: ReadonlyArray<string>
+}
+
+export class KeyboardShortcut extends React.Component<IKeyboardShortCutProps> {
+  public render() {
+    if (__DARWIN__) {
+      return this.props.darwinKeys.map((k, i) => <kbd key={k + i}>{k}</kbd>)
+    } else {
+      return this.props.keys.map((k, i) => {
+        return this.props.keys.length === i + 1 ? (
+          <kbd key={k + i}>{k}</kbd>
+        ) : (
+          <>
+            <kbd key={k + i}>{k}</kbd>
+            <>+</>
+          </>
+        )
+      })
+    }
```

Comment:
I think this is a bit cleaner, and I also added spaces around the + as we had before this PR:
```suggestion
    const keys = __DARWIN__ ? this.props.darwinKeys : this.props.keys

    return keys.map((k, i) => {
      return (
        <>
          <kbd key={k + i}>{k}</kbd>
          {!__DARWIN__ && i < keys.length - 1 ? <> + </> : null}
        </>
      )
    })
```